src/consts/index.ts

```diff
@@ -164,6 +171,11 @@ export enum Components {
   CreatePairTransactionDetails = 'TransactionDetails/CreatePairTransactionDetails',
 }
 
+export enum RewardsTabsItems {
```

Comment:
Could you please set a `PageNames `as values? To make it clear, that the tabs - it is a pages, and the values - are route names
```
export enum RewardsTabsItems {
  Rewards = PageNames.Rewards,
  ReferralProgram = PageNames.ReferralProgram
}
```